src/config/configuration.ts

```diff
@@ -1,18 +1,28 @@
+import {
+  getEnv as baseGetEnv,
+  getNodeEnv,
+  toInteger,
+} from '@fullstacksjs/toolbox';
 import { registerAs } from '@nestjs/config';
 import Joi from 'joi';
 
-interface IAppConfig {
-  nodeEnv: string;
-  port: number;
+const getEnv = baseGetEnv<Env>;
+
+export interface AppConfig {
+  nodeEnv?: string;
+  port?: number;
 }
 
 export default registerAs('app', () => {
-  const values: IAppConfig = {
-    nodeEnv: process.env['NODE_ENV'],
-    port: parseInt(process.env['PORT']),
+  const port = getEnv('PORT');
+  console.log(port);
+
+  const values: AppConfig = {
+    nodeEnv: getNodeEnv(),
+    port: port ? toInteger(port) : undefined!,
```

Comment:
`toInteger` is a friendlier version of `parseInt`, which works only for decimals.